src/tests/suite/dfuse_test.c

```diff
@@ -208,14 +208,14 @@ timespec_gt(struct timespec t1, struct timespec t2)
 void
 do_mtime(void **state)
 {
-	struct stat	stbuf;
-	struct timespec	prev_ts;
-	struct timespec	now;
-	struct timespec	times[2];
-	int		fd;
-	int		rc;
-	char		input_buf[] = "hello";
-	int		root = open(test_dir, O_PATH | O_DIRECTORY);
+	struct stat     stbuf;
+	struct timespec prev_ts;
+	struct timespec now;
+	struct timespec times[2];
+	int             fd;
+	int             rc;
+	char            input_buf[] = "hello";
+	int             root        = open(test_dir, O_PATH | O_DIRECTORY);
```

Comment:
You replaced tabs with spaces? Is this our preferred style?

Reply:
I just ran git-clang-format on it, our policy on tabs is set at https://github.com/daos-stack/daos/blob/master/.clang-format#L10

Reply:
I still wonder if that clang format file is wrong, considering we use tabs everywhere else

Reply:
When we looked at this there was a real mix with no clear style more common than another.

Reply:
Interesting... Anyway, not a blocker for me.